prac_05/word_occurences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:   25 minutes
+"""
+string = input("Text: ")
+words = string.split(" ")
+string_dictionary = {}
+longest_word_length = 0
+words.sort()
+for item in words:
+    item.lower()
+    if item in string_dictionary:
+        string_dictionary[item] += 1
+    else:
+        string_dictionary[item] = 1
+for item in string_dictionary:
```

Comment:
You might be able to use a generate function to achieve this. Something like max(len(word) for word in string_dictionary.keys()) might work?

Reply:
Note: you might think of using .keys() in the for loop. This will return a list of all the keys (names) from the dictionary